content/en/blog/2023/contributing-to-otel/index.md

```diff
@@ -0,0 +1,158 @@
+[OpenTelemetry blog](/blog). If you've done something interesting in
+OpenTelemetry, or have worked on a complex OTel implementation within your
+organization, the OpenTelemetry blog is a great way to share that with the
+community at large.
+
+Learn more about submitting a post to the OTel blog
+[here](https://github.com/open-telemetry/opentelemetry.io#submitting-a-blog-post).
+
+### **Join the OTel End User Working Group**
+
+Are you just getting started with OTel and want to connect with other OTel
+practitioners? Do you have feedback that you wish to share with the greater OTel
+community? Or perhaps you're a more advanced OTel practitioner and want to share
+your story and/or use cases with the greater community? Then the OTel End User
+Working Group (EUWG) might be just the place for you! This group is made up of
+OpenTelemetry practitioners who come together a few times a month to:
+
+- Share their feedback and ask questions through
+  [OTel End User Discussions](/community/end-user/discussion-group/)
+- Share their stories of OpenTelemetry adoption through
+  [OTel Q&A](/community/end-user/interviews-feedback/)
+- Share their knowledge on specific areas of OpenTelemetry through
+  [OTel in Practice (OTiP)](/community/end-user/otel-in-practice/)
+
+As an added bonus, I'm one of the co-chairs. Just sayin'… 😉
+
+Learn more about the OTel EUWG [here](/community/end-user/).
+
+### **Contribute to the OpenTelemetry Demo**
+
+If you'd like to contribute code, but aren't quite ready to contribute to core
+OTel code, then perhaps you might want to check out the
+[OTel Demo](/docs/demo/). The OTel Demo includes a distributed online telescope
+shop application with services written in multiple languages and is instrumented
+with OpenTelemetry. It's a great way to learn about how to instrument code in
+your chosen language.
+
+[Per the OTel Demo docs](https://github.com/open-telemetry/opentelemetry-demo#welcome-to-the-opentelemetry-astronomy-shop-demo),
+the purpose of the demo is to:
+
+- Provide a realistic example of a distributed system that can be used to
+  demonstrate OpenTelemetry instrumentation and Observability.
+- Build a base for vendors, tooling authors, and others to extend and
+  demonstrate their OpenTelemetry integrations.
+- Create a living example for OpenTelemetry contributors to use for testing new
+  versions of the API, SDK, and other components or enhancements.
+
+The OTel Demo [just celebrated its first birthday](/blog/2023/demo-birthday/),
+and it's come a long way since its first release. As OTel evolves, the OTel Demo
+must also evolve alongside it, to ensure that it is utilizing the latest and
+greatest versions of things like the [OTel Collector](/docs/collector/) and
+language-specific instrumentation. In short, there's always work to be done!
+
+Learn more about how to contribute to the OTel Demo
+[here](https://github.com/open-telemetry/opentelemetry-demo/blob/main/CONTRIBUTING.md).
+
```

Comment:
again, add your [own contributions](https://github.com/open-telemetry/opentelemetry-demo/pull/432), so people have something tangible, especially this one ticket shows that contributing is a little bit of work, but you get feedback and over some time it gets merged